lib/killbill_client/models/transaction.rb

```diff
@@ -126,7 +130,7 @@ def follow_up_path(payment_id)
       def create_initial_transaction(path, query_map, payment_method_id, user, reason, comment, options)
         query_map[:paymentMethodId] = payment_method_id unless payment_method_id.nil?
 
-        with_payment_failure_handling do
+        created_transaction = with_payment_failure_handling do
```

Comment:
While we're at it, we should also support refresh-specific options, see https://github.com/killbill/killbill-client-ruby/commit/cd225262f138b00a92215cd5e53ca76b09b6d14c.

A typical use-case is to fetch the plugin details to get the redirect information in case of 3D-S.